docs/how-certificate-issuing-works.md

```diff
@@ -11,17 +10,20 @@ To begin, the client submits a certificate request to Fulcio.
 
 ![Certificate request diagram](img/certificate-request.png)
 
-The certificate request contains three items:
+The certificate request contains the following:
 
-- An OIDC identity token. This is a JWT containing information about the
-  principal (identity of our client), the issuer (who verified this identity?
-Google, Github etc) and some other metadata.
-- The public key. This is the public half of an asymmetric key-pair generated
-  by the client. The public key will be embedded in the final x509 certificate
-if this request is successful
-- A challenge. This challenge proves the client is in possession of the private
-  key that corresponds to the public key provided. The challenge created by
-signing the subject portion of the OIDC ID token
+- An OpenID Connect (OIDC) identity token. This is a signed JWT containing information about the
+  principal (identity of the client), the issuer (who issued the identity token -
+  Google, Microsoft, GitHub, etc.) and additional metadata such as expiration.
```

Comment:
might be worth noting that the identity could be that of a developer or a workload

Reply:
done